builder/Dockerfile

```diff
@@ -6,8 +6,13 @@ RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor github.com/Azure/aks-periscope
 
 FROM alpine
 RUN apk --no-cache add ca-certificates
+RUN apk add curl openssl bash --no-cache
 ADD https://storage.googleapis.com/kubernetes-release/release/v1.16.0/bin/linux/amd64/kubectl /usr/local/bin/kubectl
 RUN chmod +x /usr/local/bin/kubectl
+RUN curl -fsSl -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 

```

Comment:
💡 Please explore the possible `featureflag` to only trigger when it is needed for the feature.

Reply:
We may download this in helm_collector collect method itself

---
In reply to: [626964750](https://github.com/Azure/aks-periscope/pull/51#discussion_r626964750) [](ancestors = 626964750)